MonoZelda/Controllers/KeyboardController.cs

```diff
@@ -2,18 +2,28 @@
 using Microsoft.Xna.Framework.Input;
 using MonoZelda.Commands;
 using System.Collections.Generic;
+using System.Linq;
 using System.Diagnostics;
 
 namespace MonoZelda.Controllers;
 
 public class KeyboardController : IController
 {
     private CommandManager _commandManager;
-    private Dictionary<(Keys key,bool oneShot), CommandType> _keyCommandDictionary;
+    private Dictionary<(Keys key, bool oneShot), CommandType> _keyCommandDictionary;
+    private List<Keys> _pressedDirectionalKeys;
+    private static readonly Keys[] _directionalKeys = new[]
+    {
+        Keys.W, Keys.Up,
+        Keys.S, Keys.Down,
+        Keys.A, Keys.Left,
+        Keys.D, Keys.Right
```

Comment:
I like the abstraction of storing directional keys in an array. Being able to use arrow keys or WASD is nice.